src/courier.js

```diff
@@ -0,0 +1,26 @@
+import EventSource from 'eventsource'
+import log from './logger'
+
+const levelFormat = {
+  debug: log.debug,
+  info: log.info,
+  warning: log.warn,
+  error: log.error,
+}
+
+export default {
+  logLevels: Object.keys(levelFormat),
+  listen: (account, workspace, level, authToken) => {
+    level = level || 'info'
+
+    let es = new EventSource(`http://courier.vtex.com/${account}/${workspace}/app-events?level=${level}`, {
+      'Authorization': `token ${authToken}`,
+    })
+    es.onopen = () => log.info(`Listening for events with level ${level}`)
+    es.addEventListener('message', (message) => {
```

Comment:
`{data}` would allow you to be more expressive in terms of what you want of this object :)